### PR TITLE
Adding Integration Test for Phone Number Authentication in Firebase

### PR DESCRIPTION
**Pull request by srushtisv**
Implement integration test for verifying phone number sign-in with Firebase Auth. Please review and approve.

### CODE REVIEW

File: FirebaseAuth/Tests/SampleSwift/SwiftApiTests/PhoneAuthTest.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
+//
```
Comment (pragatimodi) [inline FirebaseAuth/Tests/SampleSwift/SwiftApiTests/PhoneAuthTest.swift:1]: Please add Google copyrights
You can refer to https://github.com/firebase/firebase-ios-sdk/blob/main/FirebaseAuth/Tests/SampleSwift/SwiftApiTests/PhoneMultiFactorTests.swift or other files in the repo

**Comment by paulb777** [comment]
FYI, CI for the auth integration tests only run for PRs from a branch in the main firebase-ios-sdk repo in order to access the needed credentials.

File: FirebaseAuth/Tests/SampleSwift/SwiftApiTests/PhoneAuthTest.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/*
+* Copyright 2019 Google
```
Comment (pragatimodi) [inline FirebaseAuth/Tests/SampleSwift/SwiftApiTests/PhoneAuthTest.swift:2]:
```suggestion
* Copyright 2024 Google
```

**Comment by paulb777** [comment]
Replaced by #13538

**Pull request closed without merging**